### PR TITLE
Minor addition to the README

### PR DESCRIPTION
**Pull request by jgbarah**
Minor addition to the README, explaining how to migrate from the "old" MetricsGrimoire identity tables.

### CODE REVIEW

**Comment by jgbarah** [comment]
I guess this is now complete. I tested the procedure for migrating from an "old" database, and seems to work nicely.

File: README.md [hunk 1]
```diff
@@ -56,6 +56,38 @@ Configuration
   $ sortinghat init <name>
 ```
 
+Migration from "old" MetricsGrimoire identities
+-----------------------------------------------
+
+Assume we had a database named "project_scm", with information about the gi trepository of project "project", with the "old" MetricsGrimoire identities. The process to migrate to Sorting Hat is as follows (assuming sortinghat is already installed):
```
Comment (andygrunwald) [inline README.md]: `gi trepository` => `git repository`
`Sorting Hat` => `sortinghat` (it is a name)

File: README.md [hunk 2]
```diff
@@ -56,6 +56,38 @@ Configuration
...
+
+```
+[Create the Sorting Hat database for "new" identities]
```
Comment (andygrunwald) [inline README.md]: `Sorting Hat` => `sortinghat` (it is a name)

File: README.md [hunk 3]
```diff
@@ -56,6 +56,38 @@ Configuration
...
+  $ sortinghat init project_ids
+[For each data source of interest (SCM, ITS, MLS, etc.) export identities
+ to a JSON file, and import it into the Sorting Hat database]
```
Comment (andygrunwald) [inline README.md]: `Sorting Hat` => `sortinghat` (it is a name)

File: README.md [hunk 4]
```diff
@@ -56,6 +56,38 @@ Configuration
...
+  $ mg2sh -d project_scm -s project:scm -o project-scm.json
+  $ sortinghat -d project_ids load --identities  project-scm.json
+  ... [Repeat previous two step per each data source, switching "scm"
+       to "its", "mls", etc.]
+[Export all the identities in the Sorting Hat database]
```
Comment (andygrunwald) [inline README.md]: `Sorting Hat` => `sortinghat` (it is a name)

**Comment by andygrunwald** [comment]
One question: What is a "old MetricsGrimoire" database?
What version / year we are talking about here? Maybe this should be added as well to be a little more descriptive.
@sduenas Any blockers left to merge this?

**Comment by jgbarah** [comment]
I failed to address your comments, @andygrunwald, and in any case I think this is no longer relevant. So I'm closing the pr.

**Pull request closed without merging**